src/compile_cache.h

```diff
@@ -34,6 +41,7 @@ struct CompileCacheEntry {
   // Copy the cache into a new store for V8 to consume. Caller takes
   // ownership.
   v8::ScriptCompiler::CachedData* CopyCache() const;
+  const char* type_name() const;
```

Comment:
nit: `std::string_view type_name() const;` ?

Reply:
This is returning string literals for debug logs, doesn't that just adds an extra overhead for all of them?